### PR TITLE
Update AWSO CLI-based onboarding documentation

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812.

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/automatic-installation-script.md [hunk 1]
```diff
@@ -9,13 +9,24 @@ Sumo Logic provides [POSIX](https://github.com/SumoLogic/sumologic-solution-temp
...
 
 * Set up the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
 * [Configure AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) to use AWS profiles.
+* Select/Change the enabled [AWS region](https://docs.aws.amazon.com/cli/latest/reference/configure/set.html) where you want to deploy the solution.
+
+    <details><summary>Example: How to change your default AWS region to us-west-1?</summary>
+    Run the following command with AWS region as us-west-1.
+    <p></p>
+    <code>aws configure set region us-west-1</code>
+    </details>
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/automatic-installation-script.md:28]:
```suggestion
      <details><summary>Example: How to change your default AWS region to us-west-1</summary>
    Run the following command with AWS region as us-west-1:

    ```
    aws configure set region us-west-1
    ```
    </details>
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/automatic-installation-script.md [hunk 2]
```diff
@@ -9,13 +9,26 @@ Sumo Logic provides [POSIX](https://github.com/SumoLogic/sumologic-solution-temp
...
 * [Configure AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) to use AWS profiles.
+* Select/Change the enabled [AWS region](https://docs.aws.amazon.com/cli/latest/reference/configure/set.html) where you want to deploy the solution.
+
+      <details><summary>Example: How to change your default AWS region to us-west-1</summary>
+    Run the following command with AWS region as us-west-1:
+
+    ```
+    aws configure set region us-west-1
+    ```
+    </details>
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/automatic-installation-script.md:30]:
```suggestion
    <details><summary>Example: How to change your default AWS region to us-west-1</summary>
    Run the following command with AWS region as us-west-1:

    ```
    aws configure set region us-west-1
    ```
    </details>
```

**Review by kimsauce — APPROVED** [review]
I made a minor formatting fix

**Pull request merged**